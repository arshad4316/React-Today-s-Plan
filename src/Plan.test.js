import React from 'react';

import renderer from 'react-test-renderer';
import  Plan  from './Plan';

test('snapshot', () =>{
    const component = renderer.create(<Plan/>);

    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
})