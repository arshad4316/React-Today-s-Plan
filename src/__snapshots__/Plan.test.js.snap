// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Array [
  <li
    className="shadow p-2 my-2 col-sm-9"
  />,
  <button
    className="btn btn-danger my-2 col-sm-2 offset-1"
    onClick={[Function]}
  >
    X
  </button>,
]
`;
